NxMapData.lua

```diff
@@ -13283,7 +13336,14 @@ Map.MiniMapBlks = {
 		14, 16,
 		Map.MapWorldInfo[1550].X + Map.MapInfo[13].X + 594.95, Map.MapWorldInfo[1550].Y + Map.MapInfo[13].Y - 440,
 		"World\\Minimaps\\2222"
-	},	
+	},
+	--[14] = {
+	--	Map.ShadowlandsBlks,
+	--	2627,
+	--	26, 27,
+	--	Map.MapWorldInfo[1550].X + Map.MapInfo[13].X - 106.85, Map.MapWorldInfo[1550].Y + Map.MapInfo[13].Y - 295.44,
+	--	"World\\Minimaps\\2511"
+	--},	
```

Comment:
I hadn't added this back in, because I first wanted to have the base map shown before I attempt to add the blks